aws-ts-netlify-cms-and-oauth/cms/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
Let's remove this since the repo itself has a LICENSE. This is also an incorrect file as it is from the original template repo.

Reply:
Removed it